package.json

```diff
@@ -4,6 +4,7 @@
   "private": false,
   "homepage": "http://skills.susi.ai/",
   "dependencies": {
+    "@material-ui/core": "^1.4.0",
```

Comment:
Please try to implement using the same `material-ui` version that is used in the project.